### PR TITLE
Area, Point, Bubble Maps Drill Down + Invisible Links

### PR DESCRIPTION
**Pull request by kwongz**
### Checklist

- [x] I have added a [changeset](https://github.com/evidence-dev/evidence/blob/main/CONTRIBUTING.md#adding-a-changeset)
- [x] I have added to the docs where applicable

### CODE REVIEW

**Comment by kwongz** [comment]
merged feature branch in, still getting error 

image

Updated branch with pnpm install --no-frozen-lockfile, but still getting the error in the picture above

@ItsMeBrianD @zachstence

**Comment by archiewood** [comment]
@kwongz if you haven't already tried, `pnpm i` and merging the result sometimes fixes this

**Comment by zachstence** [comment]
@kwongz Looks like its working now. I just did `pnpm i --ignore-scripts` and committed/pushed the updated `pnpm-lock.yaml`.

**Comment by kwongz** [comment]
Thanks All!

File: packages/ui/core-components/src/lib/unsorted/viz/map/BaseMap.svelte [hunk 1]
```diff
@@ -74,6 +87,18 @@
 		</div>
 	</div>
 {/if}
+{#if link}
+	<QueryLoad {data} let:loaded>
+		<svelte:fragment slot="skelton"></svelte:fragment>
```
Comment (zachstence) [inline packages/ui/core-components/src/lib/unsorted/viz/map/BaseMap.svelte:92]: Is `skelton` supposed to be `skeleton`?

File: packages/ui/core-components/src/lib/unsorted/viz/map/BaseMap.svelte [hunk 2]
```diff
@@ -74,6 +87,18 @@
...
+		<InvisibleLinks data={loaded} {link} />
+	</QueryLoad>
+{/if}
+{#if linkCol}
+	<QueryLoad {data} let:loaded>
+		<svelte:fragment slot="skelton"></svelte:fragment>
```
Comment (zachstence) [inline packages/ui/core-components/src/lib/unsorted/viz/map/BaseMap.svelte:98]: Is `skelton` supposed to be `skeleton`?

File: packages/ui/core-components/src/lib/unsorted/viz/map/EvidenceMap.js [hunk 3]
```diff
@@ -199,8 +199,12 @@ export class EvidenceMap {
 						this.lastSelectedLayer = layer;
 					}
 					onclick(feature);
+				});
+				layer.on('dblclick', () => {
+					//handles map drilldowns
 					if (link) {
 						window.location.href = link;
+						this.map.doubleClickZoom.disable();
```
Comment (zachstence) [inline packages/ui/core-components/src/lib/unsorted/viz/map/EvidenceMap.js:207]: It seems a little weird to disable the `doubleClickZoom` only after the user has double clicked the map. Maybe this should happen when the map is created if they provide links?

File: packages/ui/core-components/src/lib/unsorted/viz/map/EvidenceMap.js [hunk 4]
```diff
@@ -264,9 +268,12 @@ export class EvidenceMap {
 				onclick(marker);
 			}
 			marker.bringToFront();
-
+		});
+		marker.on('dblclick', () => {
+			//handles map drilldowns
 			if (link) {
 				window.location.href = link;
+				this.map.doubleClickZoom.disable();
```
Comment (zachstence) [inline packages/ui/core-components/src/lib/unsorted/viz/map/EvidenceMap.js:276]: See other comment

File: packages/ui/core-components/src/lib/unsorted/viz/map/EvidenceMap.js [hunk 5]
```diff
@@ -301,6 +308,7 @@ export class EvidenceMap {
 	 */
 	buildTooltip(item, tooltip) {
 		let tooltipCode = '';
+		console.log(tooltip);
```
Comment (zachstence) [inline packages/ui/core-components/src/lib/unsorted/viz/map/EvidenceMap.js:311]:
```suggestion
```

**Review by zachstence — CHANGES_REQUESTED** [review]

**Comment by zachstence** [comment]
@ItsMeBrianD Do we care about this Vercel deployment? Not sure what it is or why its failing

**Comment by hughess** [comment]
@kwongz the area map click target is different than I would have expected - it seems like you need to click the specific line point rather than being able to click the area itself.

I know there's some echarts limitations with that, but is there a way to enable the clicks for the area?

**Comment by kwongz** [comment]
Sorry @hughess I think i'm misunderstanding, but on my end i can double click the areas in Area Map to navigate to the link.
I'm unclear on line point vs area? 

https://www.loom.com/share/f19742b9003b4a8eb3f2069cd6ad55c8

**Comment by hughess** [comment]
Sorry @kwongz i meant to say area _chart_!

area map is working for me

**Comment by kwongz** [comment]
@hughess, I understand now, yeah that would be a nice fix. I'll make a new issue for this since Area Charts was in another PR thats been merged in the the feature

Thanks!

**Comment by hughess** [comment]
@kwongz makes sense.

The same issue exists on the line chart, where you have to click the specific point rather than being able to click the line itself. For some line charts with a small # of points, this will cause issues for users who won't be over a point when they try to click.

Ideal would be that the click target for the line chart is the full line, and for the area it's the full area

**Pull request merged**